pkg/configfilearg/parser_test.go

```diff
@@ -150,14 +150,14 @@ func TestConfigFile(t *testing.T) {
 
 func TestParse(t *testing.T) {
 	testDataOutput := []string{
-		"--foo-bar", "baz",
-		"--a-slice", "1",
-		"--a-slice", "2",
-		"--a-slice", "",
-		"--a-slice", "three",
+		"--foo-bar=baz",
+		"--a-slice=1",
+		"--a-slice=2",
+		"--a-slice=",
+		"--a-slice=three",
 		"--isempty",
-		"-c", "b",
-		"--islast", "true",
+		"-c=b",
+		"--islast=true",
```

Comment:
should you add a test for a bool flag with a false value?

Reply:
Will do.

Reply:
Added